editorconfig-eclipse-functional-test/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: EditorConfig Eclipse Functional Test
 Bundle-SymbolicName: editorconfig-eclipse-functional-test;singleton:=true
-Bundle-Version: 0.3.0.qualifier
+Bundle-Version: 0.4.0.qualifier
```

Comment:
This was overlooked. I've already pushed this change to master and updated the "update-versions.sh" script so that this doesn't happen again.